--- origsrc/qt-x11-free-3.3.8/src/moc/moc.y	2007-02-02 08:01:02.000000000 -0600
+++ src/qt-x11-free-3.3.8/src/moc/moc.y	2007-10-28 22:15:48.328125000 -0500
@@ -2359,7 +2359,7 @@
 	bool hasReturnValue = f->type != "void" && (validUType( f->type ) || isVariantType( f->type) );
 
 	if ( hasReturnValue || !f->args->isEmpty() ) {
-	    fprintf( out, "    static const QUParameter param_%s_%d[] = {\n", functype, list->at() );
+	    fprintf( out, "    static QUParameter param_%s_%d[] = {\n", functype, list->at() );
 	    if ( hasReturnValue ) {
 		if ( validUType( f->type ) )
 		    fprintf( out, "\t{ 0, &static_QUType_%s, %s, QUParameter::Out }", uType(f->type).data(), uTypeExtra(f->type).data() );
@@ -2387,7 +2387,7 @@
 	    fprintf( out, "\n    };\n");
 	}
 
-	fprintf( out, "    static const QUMethod %s_%d = {", functype, list->at() );
+	fprintf( out, "    static QUMethod %s_%d = {", functype, list->at() );
 	int n = f->args->count();
 	if ( hasReturnValue )
 	    n++;
@@ -2415,7 +2415,7 @@
 	f->signature += ")";
     }
     if ( list->count() ) {
-	fprintf(out,"    static const QMetaData %s_tbl[] = {\n", functype );
+	fprintf(out,"    static QMetaData %s_tbl[] = {\n", functype );
 	f = list->first();
 	while ( f ) {
 	    fprintf( out, "\t{ \"%s\",", f->signature.data() );
@@ -2462,7 +2462,7 @@
     fprintf( out, "#ifndef QT_NO_PROPERTIES\n" );
     int i = 0;
     for ( QPtrListIterator<Enum> it( g->enums ); it.current(); ++it, ++i ) {
-	fprintf( out, "    static const QMetaEnum::Item enum_%i[] = {\n", i );
+	fprintf( out, "    static QMetaEnum::Item enum_%i[] = {\n", i );
 	int k = 0;
 	for( QStrListIterator eit( *it.current() ); eit.current(); ++eit, ++k ) {
 	    if ( k )
@@ -2471,7 +2471,7 @@
 	}
 	fprintf( out, "\n    };\n" );
     }
-    fprintf( out, "    static const QMetaEnum enum_tbl[] = {\n" );
+    fprintf( out, "    static QMetaEnum enum_tbl[] = {\n" );
     i = 0;
     for ( QPtrListIterator<Enum> it2( g->enums ); it2.current(); ++it2, ++i ) {
 	if ( i )
@@ -2734,7 +2734,7 @@
 
 	fprintf( out, "#ifndef QT_NO_PROPERTIES\n" );
 
-	fprintf( out, "    static const QMetaProperty props_tbl[%d] = {\n ", g->props.count() );
+	fprintf( out, "    static QMetaProperty props_tbl[%d] = {\n ", g->props.count() );
 	for( QPtrListIterator<Property> it( g->props ); it.current(); ++it ) {
 
 	    fprintf( out, "\t{ \"%s\",\"%s\", ", it.current()->type.data(), it.current()->name.data() );
@@ -2814,7 +2814,7 @@
 	moc_err("The declaration of the class \"%s\" contains class infos"
 		" but no Q_OBJECT macro.", g->className.data());
 
-    fprintf( out, "    static const QClassInfo classinfo_tbl[] = {\n" );
+    fprintf( out, "    static QClassInfo classinfo_tbl[] = {\n" );
     int i = 0;
     for( QPtrListIterator<ClassInfo> it( g->infos ); it.current(); ++it, ++i ) {
 	if ( i )
--- origsrc/qt-x11-free-3.3.8/tools/designer/plugins/glade/glade2ui.cpp	2007-01-11 07:47:11.000000000 -0600
+++ src/qt-x11-free-3.3.8/tools/designer/plugins/glade/glade2ui.cpp	2007-10-28 22:15:48.375000000 -0500
@@ -40,7 +40,7 @@
 
 #include <ctype.h>
 
-static const struct {
+static struct {
     const char *gtkName;
     const char *qtName;
 } classNames[] = {
@@ -94,7 +94,7 @@
     { 0, 0 }
 };
 
-static const struct {
+static struct {
     const char *name;
     const char *menuText;
 } stockMenuItems[] = {
@@ -133,7 +133,7 @@
     { 0, 0 }
 };
 
-static const struct {
+static struct {
     const char *gtkName;
     int qtValue;
 } keys[] = {
