--- origsrc/qt-x11-free-3.3.8/config.tests/unix/checkavail	2003-05-27 10:19:12.000000000 -0500
+++ src/qt-x11-free-3.3.8/config.tests/unix/checkavail	2007-10-28 22:15:48.250000000 -0500
@@ -9,9 +9,9 @@
 MODULE_NAME=$1
 VERBOSE=$2
 shift 2
-LIBDIRS="/lib /usr/lib"
+LIBDIRS="/lib /usr/lib /usr/lib/w32api"
 LIBNAMES=""
-INCLUDEDIRS="/usr/include"
+INCLUDEDIRS="/usr/include /usr/include/w32api"
 INCLUDES=""
 
 PARAMS=$@
--- origsrc/qt-x11-free-3.3.8/configure	2007-01-15 06:41:50.000000000 -0600
+++ src/qt-x11-free-3.3.8/configure	2007-10-28 22:15:48.265625000 -0500
@@ -1623,7 +1623,7 @@
     MYSQL_REQ="-lmysqlclient mysql.h"
     OCI_REQ="-lclntsh -lwtc8 oci.h"
     PSQL_REQ="-lpq postgres.h libpq/libpq-fs.h catalog/pg_type.h libpq-fe.h"
-    ODBC_REQ="-lodbc sql.h sqlext.h"
+    ODBC_REQ="-lodbc32 sql.h sqlext.h"
     TDS_REQ="-lsybdb sybfront.h sybdb.h"
     DB2_REQ="-ldb2 sqlcli.h sqlcli1.h"
     IBASE_REQ="-lgds ibase.h"
--- origsrc/qt-x11-free-3.3.8/plugins/src/sqldrivers/odbc/odbc.pro	2003-12-08 06:29:45.000000000 -0600
+++ src/qt-x11-free-3.3.8/plugins/src/sqldrivers/odbc/odbc.pro	2007-10-28 22:15:48.281250000 -0500
@@ -17,7 +17,8 @@
 unix {
 	OBJECTS_DIR	= .obj
 	!contains( LIBS, .*odbc.* ) {
-	    LIBS 	*= -lodbc
+		!cygwin-g++:LIBS *= -lodbc
+	    cygwin-g++:LIBS	*= -lodbc32
 	}
 }
 
--- origsrc/qt-x11-free-3.3.8/src/sql/drivers/odbc/qsql_odbc.h	2007-01-11 07:38:34.000000000 -0600
+++ src/qt-x11-free-3.3.8/src/sql/drivers/odbc/qsql_odbc.h	2007-10-28 22:15:48.343750000 -0500
@@ -48,6 +48,11 @@
 #include <qt_windows.h>
 #endif
 
+#if defined (Q_OS_CYGWIN)
+#define WIN32_LEAN_AND_MEAN
+#include <windows.h>
+#endif
+
 #if defined (Q_OS_MAC)
 // assume we use iodbc on MAC
 // comment next line out if you use a 
--- origsrc/qt-x11-free-3.3.8/src/sql/drivers/odbc/qsql_odbc.cpp	2007-01-11 07:38:34.000000000 -0600
+++ src/qt-x11-free-3.3.8/src/sql/drivers/odbc/qsql_odbc.cpp	2007-10-28 22:15:48.343750000 -0500
@@ -40,6 +40,8 @@
 #if defined (Q_OS_WIN32)
 #include <qt_windows.h>
 #include <qapplication.h>
+#elif defined (Q_OS_CYGWIN)
+#undef BOOL
 #endif
 #include <qdatetime.h>
 #include <private/qsqlextension_p.h>
