--- origsrc/qt-x11-free-3.3.8/config.tests/x11/xfreetype.test	2004-08-09 15:16:57.000000000 -0500
+++ src/qt-x11-free-3.3.8/config.tests/x11/xfreetype.test	2007-10-28 22:15:48.250000000 -0500
@@ -41,7 +41,7 @@
 LIBS="Xft2 Xft"
 for LIB in $LIBS; do
     for LIBDIR in $LIBDIRS; do
-	EXTENSIONS="a so sl"
+	EXTENSIONS="a dll.a so sl"
 	for E in $EXTENSIONS; do
 	    if [ -f "$LIBDIR/lib$LIB.$E" ]; then
 		F="$LIB"
--- origsrc/qt-x11-free-3.3.8/configure	2007-01-15 06:41:50.000000000 -0600
+++ src/qt-x11-free-3.3.8/configure	2007-10-28 22:15:48.265625000 -0500
@@ -2396,7 +2396,7 @@
 	else
 	    echo "Do you accept the terms of $affix license? \c"
 	fi
-	read acceptance
+	acceptance=yes
 	echo
 	if [ "$acceptance" = "yes" ]; then
 	    break
